dev/page-building-studio/schemaTypes/hero.ts

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
I'm unsure if it's OK to suppress this error. I've only installed `react-icons` in this workspace.